pyproject.toml

```diff
@@ -13,4 +13,9 @@ requires-python = ">=2.7,!=3.0.*,!=3.1.*,!=3.2.*,!=3.3.*,!=3.4.*"
 requires = [
   "configparser >= 3.5; python_version < '3'",
   "importlib-resources; python_version < '3.7'",
+  "build >= 0.2.0",  # not a hard runtime requirement -- we can softfail
+  "packaging",  # not a hard runtime requirement -- we can softfail
```

Comment:
And... assuming that we're reducing scope, we don't need either dependency.

Reply:
If we end up expanding the scope of `installer` to also handle builds+installs from source we probably should namespace vendor these as `build` would otherwise effectively introduce a circular dependency in `installer` otherwise.

Reply:
That’s not gonna happen — this isn’t a drop in replacement package for pip. If you want that, please feel free to write a package for that.